### PR TITLE
Build on localhost

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,19 @@
...
-An apt- or pacman-based Linux distribution.
+An apt- or pacman-based Linux distribution on both the control machine and the targets.
+If another distribution is used, the packages for `git`, `go`, and `make` need to be installed on the control machine and `go` needs to be installed on the targets.
 
 ## Role Variables
 
 | Name                    | Required                 | Default           | Description                                                              |
 |-------------------------|:------------------------:|-------------------|--------------------------------------------------------------------------|
+| `global_cache_dir`      | :heavy_check_mark:       |                   | Path where files are downloaded and Consul is built.                     |
 | `consul_version`        | :heavy_check_mark:       | `0.7.5`           | Version of Consul to build and run.                                      |
```
Comment (SlothOfAnarchy) [inline README.md]: fix your inconsistent use of default values and the required flag, I've told you this before.

File: tasks/build.yml [hunk 2]
```diff
@@ -1,27 +1,76 @@
 ---
-- name: Install dependencies (apt)
+- name: Gather localhost's facts
+  action: setup
+  delegate_facts: True
+  delegate_to: localhost
+
+- name: Install build dependencies (apt)
```
Comment (SlothOfAnarchy) [inline tasks/build.yml]: add tasks to update package cache

File: tasks/build.yml [hunk 3]
```diff
@@ -1,27 +1,76 @@
...
     - git
     - go
     - make
+  when: "hostvars['localhost']['ansible_pkg_mgr'] == 'pacman'"
+  delegate_to: localhost
+
+- name: Install run dependencies (apt)
+  apt:
+    name: "{{ item }}"
+  with_items:
```
Comment (SlothOfAnarchy) [inline tasks/build.yml]: Unnecessary iteration for one item. Same case below.

Comment (dasJ) [inline tasks/build.yml]: Did it for consistency, but will change

File: README.md (hunk 1 above)
Comment (dasJ) [inline README.md]: Not related to this PR

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

File: README.md (hunk 1 above)
Comment (SlothOfAnarchy) [inline README.md]: You can fix it while your at it.

File: tasks/build.yml (hunk 2 above)
Comment (SlothOfAnarchy) [inline tasks/build.yml]: I'm surprised you actually did that!

File: README.md (hunk 1 above)
Comment (dasJ) [inline README.md]: No.

**Pull request merged**